store/tikv/region_cache.go

```diff
@@ -305,10 +307,29 @@ func (c *RegionCache) asyncCheckAndResolveLoop() {
 		case <-c.notifyCheckCh:
 			needCheckStores = needCheckStores[:0]
 			c.checkAndResolve(needCheckStores)
+		case <-ticker.C:
+			// refresh store once a minute to update labels
+			c.markAllNeedCheck()
```

Comment:
simpler directly`reResolve` all stores.

Reply:
I think we should reuse `needCheck ` state

Reply:
IMO, `needCheck` is a way fast update `on failed` in before. for now, periodic update labels is a maintenance behavior，reuse `needCheck` make more complicated.

Reply:
That's ok to me. updated.